build/nested-cluster/BUILD

```diff
@@ -0,0 +1,49 @@
+package(default_visibility = ["//visibility:public"])
+
+load("@io_bazel_rules_docker//docker:docker.bzl", "docker_build", "docker_bundle")
+
+docker_build(
+    name = "nested-kubelet-build",
+    base = "@nested-kubelet-base//image",
+    debs = [
+        "//build/debs:kubelet",
+        "//build/debs:kubectl",
+        "//build/debs:kubeadm",
+    ],
+    entrypoint = ["/init-wrapper.sh"],
+    files = [
+        "init-wrapper.sh",
+        "start.sh",
+        "//build:master-components-amd64",
+    ],
+)
+
+docker_bundle(
+    name = "nested-kubelet-bundle",
+    images = {"gcr.io/google-containers/nested-kubelet:0.1": ":nested-kubelet-build"},
+)
+
+docker_build(
+    name = "encapsulated-cluster-build",
+    base = "nested-kubelet-build",
+    files = [":nested-kubelet-bundle.tar"],
+)
+
+docker_bundle(
+    name = "encapsulated-cluster-bundle",
+    images = {"gcr.io/google-containers/encapsulated-cluster:0.1": "encapsulated-cluster-build"},
```

Comment:
do we intend to push this to gcr.io? or just build + run locally? (I don't understand the version tag "0.1" here.)

Reply:
It should be possible to build this, push it to gcr, and pull it down elsewhere to actually run the tests. It's also possible people might be interested in using this to test something on kubernetes.

I don't plan on either of those immediately, but it seems an easy way to keep it from becoming difficult.